### PR TITLE
fix(ru): image and typo

### PR DESCRIPTION
**Pull request by placidp**
### Description
1. Current background-image: ('https://lorempixel.com/400/200/') is not working, replaced it with one from en docs.
2. Fixed typo at div.container

### CODE REVIEW

File: files/ru/web/css/backdrop-filter/index.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ body {
 }
 
 body {
-  background-image: url('https://lorempixel.com/400/200/');
+  background-image: url('https://yari-demos.prod.mdn.mozit.cloud/en-US/docs/Web/CSS/backdrop-filter/anemones.jpg');
```
Comment (Saionaro) [inline files/ru/web/css/backdrop-filter/index.md:78]: @placidp please upload `anemones.jpg` to `files/ru/web/css/backdrop-filter/` directory to have an ability to use relative path.

**Review by saionaro — CHANGES_REQUESTED** [review]
@placidp thank you for contributing! Please, take a look at the my comment

File: files/ru/web/css/backdrop-filter/index.md (hunk 1 above)
Comment (placidp) [reply files/ru/web/css/backdrop-filter/index.md]: > @placidp please upload `anemones.jpg` to `files/ru/web/css/backdrop-filter/` directory to have an ability to use relative path.

Thanks for your response! Please take a look at my changes.

**Review by saionaro — APPROVED** [review]
@placidp LGTM, thank you!

**Pull request merged**